madgraph5amcatnlo.spec

```diff
@@ -35,6 +35,18 @@ rm bin/compile.py
 # Save patched config
 mv input/mg5_configuration_patched.txt input/mg5_configuration.txt
 
+# Start small NLO event generation to make sure that all additional packages are compiled
+cat <<EOF > basiceventgeneration.txt
```

Comment:
hi- @alberto-sanchez - i find this bit of proposed code to be crazy. What is the problem its trying to solve? I would think we could get a proper library build from the authors..

Reply:
Dear @Andrej-CMS, Any news about H7?

Reply:
@alberto-sanchez 
I'm implementing Herwig 7.1.1. and found a bug in combination of Herwig 7.1.1. with Madgraph. I'm in contact with the authors in regards to this. The bug also exists in the standalone Herwig 7.1.1 installation, so it's not a result of the CMSSW implementation. As soon as Herwig 7.1.1 can be interfaced correctly with MG5aMC@NLO, I will update the PR.

Kind regards,
Andrej